substrate/utils/prometheus/src/lib.rs

```diff
@@ -98,46 +98,49 @@ async fn init_prometheus_with_listener(
 	listener: tokio::net::TcpListener,
 	registry: Registry,
 ) -> Result<(), Error> {
-	let listener = hyper::server::conn::AddrIncoming::from_listener(listener)?;
-	log::info!("〽️ Prometheus exporter started at {}", listener.local_addr());
-
-	let service = make_service_fn(move |_| {
-		let registry = registry.clone();
-
-		async move {
-			Ok::<_, hyper::Error>(service_fn(move |req: Request<Body>| {
-				request_metrics(req, registry.clone())
-			}))
-		}
-	});
+	log::info!(target: "prometheus", "〽️ Prometheus exporter started at {}", listener.local_addr()?);
 
-	let (signal, on_exit) = tokio::sync::oneshot::channel::<()>();
```

Comment:
I have removed this because there is no way to stop the server..... and having graceful shutdown doesn't make sense anymore.

The way it works is that we just drop the tokio executor and doesn't gracefully shutdown this anyway